celeste/src/login/login_util.rs

```diff
@@ -88,14 +88,6 @@ pub fn username_input() -> EntryRow {
     let input = EntryRow::builder().title("Username").build();
     input.connect_changed(|input| {
         let _text = input.text();
-
-        if input.text().contains(':') {
-            input.add_css_class("error");
-            input.set_tooltip_text(Some("Username isn't allowed to contain ':'."));
-        } else {
-            input.remove_css_class("error");
-            input.set_tooltip_text(None);
-        }
     });
     input
 }
```

Comment:
Just realized it, but we can also completely remove these `.connect_changed()` calls because we don't do anything in them anymore. We should just be able to return the `EntryRow` directly.